Sources/Tests/UnitTests/Calculus/SeriesTest.cs

```diff
@@ -19,6 +21,27 @@ public void TaylorDirect(string funcOverTRaw, string expectedRaw, int termCount)
             Assert.Equal(expected, actual);
         }
 
+        [Theory]
+        [InlineData("a t3 + b t2 + c t + d", "((d) + ((c) * (x))) + (((((2) * (b)) * ((x) ^ (2))) / ((2)!)) + ((((6) * (a)) * ((x) ^ (3))) / ((3)!)))", 4)]
+        [InlineData("(e^t)ln(1+s)", "(0) + ((y) + ((((2) * ((x) * (y))) + ((-1) * ((y) ^ (2)))) / ((2)!)))", 3)]
+        [InlineData("sin(t)+cos(s)+tan(u)", "((1) + ((x) + (z)))", 2)]
+        [InlineData("cos(t)sin(s)", "((0) + (y)) + ((0) + ((((-3) * (((x) ^ (2)) * (y))) + ((-1) * ((y) ^ (3)))) / ((3)!)))", 4)]
+        [InlineData("sin(t)sin(s)sin(u)sin(v)", "((0) + (0)) + ((0) + ((0) + (((24) * ((((x) * (y)) * (z)) * (w))) / ((4)!))))", 5)]
+        public void MultivariableTaylorDirect(string funcOverTRaw, string expectedRaw, int termCount)
```

Comment:
See the previous comment, we still need more tests. For single variable you can copy the tests from single-variable taylor expansion. A one-term test should be trivial